components/Cards/CardsList.tsx

```diff
@@ -19,7 +19,7 @@ const CardsList: FC<{ cards: IData[] }> = ({ cards }) => {
 
   return (
     <>
-      <ul className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4 justify-items-stretch ">
+      <ul className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4 justify-items-stretch border-dark-silver border-2 border-opacity-75 shadow-lg shadow-silver">
```

Comment:
@CBID2, here is the fix.

```suggestion
      <ul className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4 justify-items-stretch">
```